yt_dlp/extractor/floatplane.py

```diff
@@ -124,6 +186,23 @@ def _real_extract(self, url):
         if not any(traverse_obj(post_data, ('metadata', ('hasVideo', 'hasAudio')))):
             raise ExtractorError('Post does not contain a video or audio track', expected=True)
 
+        uploader_url = format_field(
+            post_data, [('creator', 'urlname')], 'https://www.floatplane.com/channel/%s/home') or None
+        channel_url = urljoin(f'{uploader_url}/', traverse_obj(post_data, ('channel', 'urlname')))
+
+        common_info = {
+            **traverse_obj(post_data, {
+                'uploader': ('creator', 'title'),
+                'uploader_id': ('creator', 'id'),
+                'channel': ('channel', 'title'),
+                'channel_id': ('channel', 'id'),
```

Comment:
I know this wasn't there initially, but to be safe we might as well add:
```suggestion
                'uploader': ('creator', 'title', {str}),
                'uploader_id': ('creator', 'id', {str}),
                'channel': ('channel', 'title', {str}),
                'channel_id': ('channel', 'id', {str}),
```